app/views/works/_s3_resources.html.erb

```diff
@@ -14,29 +16,32 @@
           <tbody>
             <% if @work.uploads.empty? %>
               <tr class="files">
-                <td><span><%= t('works.form.curation_uploads.empty') %></span></td>
+                <td><span>No files in S3</span></td>
```

Comment:
Curious why you remove the translation here?  Not a blocker at all...

Reply:
Good catch. Restored.